packages/@okta/vuepress-site/docs/index.md

```diff
@@ -1,111 +0,0 @@
----
-# This file controls the links and content shown on the /docs landing page.
-
-component: Documentation
```

Comment:
See if the vue component can be deleted, too. It's probably the only page that uses it